src/main/java/org/circle8/dao/PuntoReciclajeDao.java

```diff
@@ -21,7 +25,22 @@
 
 import com.google.inject.Inject;
 
-public class PuntoReciclajeDao extends Dao{
+public class PuntoReciclajeDao extends Dao {
+
+	private static final String SELECT_TIPOS = """
+		   SELECT tr."ID", tr."Nombre"
+		     FROM public."TipoResiduo" AS tr
+		    WHERE 1=1
+		""";
+
+	private static final String WHERE_NOMBRE_TIPOS = """
+			AND tr."Nombre" IN (%s)
+		""";
+
+	private static final String INSERT_SQL = """
+      INSERT INTO public."PuntoReciclaje"("CiudadanoId", "Latitud", "Longitud", "DiasAbierto", "Titulo")
+  		VALUES (?, ?, ?, ?, ?)
+  		""";
```

Comment:
Tienen mezcla de espacios y tabs
```suggestion
		INSERT INTO public."PuntoReciclaje"("CiudadanoId", "Latitud", "Longitud", "DiasAbierto", "Titulo")
		VALUES (?, ?, ?, ?, ?)
		""";
```